spec/requests/landing_page_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
 
-describe 'Home page', type: :request do
+describe 'Landing page', type: :request do
   it 'displays information about this project' do
     visit root_url
-    expect(page).to have_content('Home Page')
+    expect(page).to have_content('Project Give Back')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.